jbpm-wb-integration/jbpm-wb-integration-client/src/main/java/org/jbpm/workbench/wi/client/workitem/ServiceTasksRepositoryListPresenter.java

```diff
@@ -60,12 +66,30 @@ public void init() {
         retrieveConfig();
         refreshData();
     }
+
+    @Inject
+    public void setConfirmPopup(final ConfirmPopup confirmPopup){
+        this.confirmPopup = confirmPopup;
+    }
       
     @WorkbenchPartView
     public ServiceTasksRepositoryListView getView() {
         return view;
     }
 
+    public ServiceTasksRepositoryListPresenter(){
```

Comment:
move constructor right after the fields so that we follow Class structure: 1.fields, 2. constructors, 3. methods

Reply:
+1